Tests/Functional/ControllerResolverTest.php

```diff
@@ -67,4 +69,19 @@ public function testAnnotationControllerServiceExtendingClassicService()
 
         $this->assertEquals('hello', $client->getResponse()->getContent());
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testInvokableControllerAsService()
+    {
+        if (version_compare(Kernel::VERSION_ID, '20600') < 0) {
```

Comment:
Thanks for the explanation @adam187. 
I would disable param converters in our functional tests as we don't use them instead of removing a test. 
(https://github.com/schmittjoh/JMSDiExtraBundle/blob/master/Tests/Functional/config/default . yaml should have a section configuring SensioFrameworkExtraBundle)